i18n/translations/en.json

```diff
@@ -9,6 +9,7 @@
   "collapse": "Collapse",
   "collapse.all": "Collapse All",
   "copy": "Copy",
+  "copy.all": "Copy all",
```

Comment:
I guess we don't need this anymore?

Reply:
Yes, I think we can remove it.

Reply:
I think we should use it in the field dropdown. Only "copy" doesn't help that much if it's not tied to a specific block

Reply:
Since it is in the options of the block field, I think we can only use `copy`, but of course `copy all` is more understandable.

Reply:
Ok, let's keep Copy all in the main dropdown and Copy in the block dropdown.